templates/CRM/Contact/Import/Form/Summary.tpl

```diff
@@ -16,10 +16,9 @@
  {include file="CRM/common/WizardHeader.tpl"}
  <div class="help">
    <p>
-     {if $unprocessedRowCount}
-       <strong>{ts}The import is still processing.{/ts}</strong>
-     {else}
-       <strong>{ts}Import has completed successfully.{/ts}</strong>
+     <strong>{ts 1=$statusLabel}Import Status: %1{/ts}</strong>
```

Comment:
@eileenmcnaughton so instead of the 2 pre-written messages, this switches to showing the status label. Is that ok?

Reply:
@colemanw is the status label translated?

Reply:
@eileenmcnaughton it should be. It is an option value label.